app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics");
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.all("/*", (req, res) => {
+  res.status(404).send({ message: "Not Found" });
+});
+
```

Comment:
Think about putting your final 500: server error in here at some point - maybe in a future commit